firmware/arduino/chessboard/chessboard.ino

```diff
@@ -213,6 +230,18 @@ void loop()
         sentMessage[2] = moveCount;
         sendMessageToPi(sentMessage);
 
+        // Print outgoing message
+        if (DEBUG)
+        {
+          Serial.println("Outgoing Message:  ");
+          Serial.print(sentMessage[0]);
```

Comment:
These all say message[0], need to be 0 1 2. Please update the others too.

Reply:
Actually should this printout just be in the sendMessageToPi function so that we don’t have this duplicated code at all the function calls?